### PR TITLE
Add support for ,AOE (Dvorak) key navigation in the flame chart

### PR DESCRIPTION
**Pull request by kenzieschmoll**
Fixes https://github.com/flutter/devtools/issues/2763. @Hixie

### CODE REVIEW

**Comment by polina-c** [comment]
Can you add video to demonstrate UX?
Should we link documentation/explanation for those who do not know what is AOE?

**Comment by Hixie** [comment]
woo!

**Comment by Hixie** [comment]
@polina-c `,AOE` are the four keys in the same position in a Dvorak key layout as `WASD` on a QWERTY key layout.
https://en.wikipedia.org/wiki/Dvorak_keyboard_layout

**Review by CoderDake — APPROVED** [review]

File: packages/devtools_app/lib/src/shared/charts/flame_chart.dart [hunk 1]
```diff
@@ -455,10 +455,11 @@ abstract class FlameChartState<T extends FlameChart,
   KeyEventResult _handleKeyEvent(RawKeyEvent event) {
     // Only handle down events so logic is not duplicated on key up.
     if (event is RawKeyDownEvent) {
-      // Handle zooming / navigation from W-A-S-D keys.
+      // Handle zooming / navigation from WASD keys or ,AOE keys (Dvorak users)
       // TODO(kenz): zoom in/out faster if key is held. It actually zooms slower
       // if the key is held currently.
-      if (event.logicalKey == LogicalKeyboardKey.keyW) {
+      if (event.logicalKey == LogicalKeyboardKey.keyW ||
+          event.logicalKey == LogicalKeyboardKey.comma) {
```
Comment (bkonyi) [inline packages/devtools_app/lib/src/shared/charts/flame_chart.dart:462]: Does this mean that ,OE will also allow for navigation by QWERTY users? If so, we should add a setting that allows for this to be toggled if we're unable to detect keyboard layout.

**Review by bkonyi — CHANGES_REQUESTED** [review]

**Pull request merged**